src/index.ts

```diff
@@ -1,6 +1,7 @@
 /*
 The SDK interface for games to interact with Rune.
 */
+import {setup} from './internal';
```

Comment:
We have other internal functions (see e.g. `_randomNumberGenerator` below) that we may wanna move out into a separate file. Perhaps it'd be cleaner to name this file as `browser`, `internal/browser`, or similar? :)

Reply:
I've renamed the file/function to `setupBrowser` for now.